### ISSUE TITLE
Remove XHR for Axios

### ISSUE THREAD

**Issue by PTZ2002** [keystonejs/keystone#4577]
Hello,
I'm thinking to upgrade the v4.0.0-beta.8 to support Axios and not to use Xhr.

Upgrading to axios I think will solve some general bugs in the current keystone like:  headers, contentType and support for IE to.

@JedWatson @mxstbr @autoboxer  ?

**Comment by laurenskling** [comment]
I would choose to upgrade to the `fetch` protocol, not to use XHR wrappers.

**Comment by PTZ2002** [comment]
I was thinking on Axios and a fallback to fetch for browsers that not support it

What do you say @laurenskling ?

**Comment by laurenskling** [comment]
still you can use fetch:
https://www.npmjs.com/package/whatwg-fetch
https://www.npmjs.com/package/isomorphic-fetch

**Comment by PTZ2002** [comment]
@laurenskling thank you, but what do you know about IE support?

![screenshot from 2018-02-21 15-01-11](https://user-images.githubusercontent.com/5390741/36482058-a63104f4-171a-11e8-9381-725d6365a386.png)

**Comment by laurenskling** [comment]
^ these packages will polyfill for IE

**Comment by PTZ2002** [comment]
@Noviny  @autoboxer  @mxstbr  any suggestions ? 
I have made a research on @laurenskling idea and I think his solution feets better then axios.

**Comment by autoboxer** [comment]
@PTZ2002, I agree that @laurenskling's push for **fetch** is the right approach.  Moving towards implementations that are EcmaScript standards and will be natively supported is, in my eyes, always the preferred route.  @PTZ2002, when you created this issue, were there specific bugs you had in mind related to headers, contentType and support for IE?  Referencing any bugs that this might resolve will help the case for making any needed change.

**Comment by PTZ2002** [comment]
Here are some issues that already where listed with XHR #3545   #3337  #3734
